package.json

```diff
@@ -23,14 +23,19 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint src/**/*"
+    "lint": "eslint src/**/*.js"
   },
   "dependencies": {
+    "@material-ui/core": "^4.1.2",
+    "@material-ui/icons": "^4.2.1",
     "axios": "^0.19.0",
+    "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-map-gl": "^5.0.3",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.0.1",
+    "react-select": "^3.0.4",
     "semantic-ui-react": "^0.87.2",
```

Comment:
I'm noticing that you're using _both_ semantic UI and material UI.

Reply:
Have tried some container from material or vanilla javascript that don't play well with position absolute for background map. I will try to remove it if I find something that can replace <Segment> from semantic.